proto/proto-backplane-grpc/build.gradle

```diff
@@ -1,5 +1,3 @@
-import nl.javadude.gradle.plugins.license.License
```

Comment:
It looks like the license manager gets dropped here.  What is going on?

Reply:
OK.  Please make certain the new infrastructure for licenses is in place before the next release.